daemons/attrd/pacemaker-attrd.c

```diff
@@ -333,15 +333,18 @@ attrd_cluster_connect()
     return pcmk_ok;
 }
 
-/* *INDENT-OFF* */
-static struct crm_option long_options[] = {
-    /* Top-level Options */
-    {"help",    0, 0, '?', "\tThis text"},
-    {"verbose", 0, 0, 'V', "\tIncrease debug output"},
-
-    {0, 0, 0, 0}
+static pcmk__cli_option_t long_options[] = {
+    // long option, argument type, storage, short option, description, flags
+    {
+        "help",     no_argument, NULL, '?',
+        "\tThis text", pcmk__option_default
+    },
+    {
+        "verbose",  no_argument, NULL, 'V',
+        "\tIncrease debug output", pcmk__option_default
+    },
+    { 0, 0, 0, 0 }
```

Comment:
I'm not really up on the intricacies of defining C structures, but should the end one be six 0s or will the compiler know what to do if there's only four?  And if that's the case, would the compiler do the right thing if it were just one 0?  This would appear to affect all changed files.

Reply:
Yep anything left unspecified at the end will be zeroed, so a single 0 would be sufficient. I'm guessing it's 4 b/c that was originally the number of struct members. One or all would be better, but I'm inclined not to spend the time on it since it works and we'll eventually replace it with GOption anyway